cylc/flow/time_parser.py

```diff
@@ -355,78 +359,69 @@ def _get_interval_from_expression(
     def _get_min_from_expression(
         self,
         expr: str,
-        context: 'TimePoint'
+        context: Optional['TimePoint']
     ) -> str:
```

Comment:
A docstring might have been nice?

Reply:
(Sighing) Yes it would have. When someone figures out what this is doing it would also be a good idea to add a unit test

Reply:
Out of scope to be fair.